src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private int index = new Random().nextInt(Colors.values().length);
+
     public String getRandomColor() {
```

Comment:
```suggestion
public class ColorSupplier {
    private Random random = new Random();

    public String getRandomColor() {
    int index = rendom.nextInt(Colors.values().length);
    return Colors.values()[index].name();
```

я имела ввиду вот так)

Reply:
Ааа) Все я понял) Спасибо